duckduckgo/duckduckgo-privacy-extension
# new translations for short description with privacy permissions message

## Description:

Added translated short description with additional locales.
These show up in the chrome store descriptions themselves, not the extension.

Including the new message about permissions.
code to create this stuff will eventually be added as a separate pr.

## How to test

two-parter.

1. should load locally; manifest or locale file errors should show up here (unclear if it tests them)
2. when uploading final zip file to the chrome store, all the translations (including new ones) should show up in the UI.

If there are errors they might not show up until the upload, so this PR might need to be fixed at that point. But this went smoothly the first time around - this is just an update with some new ones.

3. Long descriptions, if available, will have to be pasted one by one (defaults to the default local tho, English)

should load locally; localized 


## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
